### PR TITLE
Update privacy-notice.md

### PR DESCRIPTION
**Pull request by JanePorter**
Update to privacy notice for google analytics

### CODE REVIEW

File: content/legal-notices/privacy-notice.md [hunk 1]
```diff
@@ -95,8 +95,29 @@ We use the following types of cookies on our websites and online services:
 
 You can choose how we use functional and advertising cookies by changing the cookie settings. To do so, please go to "Cookie preferences" at the bottom of this page. In the cookie settings, you will also find an overview of the third parties that place cookies on our websites. In addition, you can control and restrict the placement of cookies via the browser settings. In this context, you can also delete existing cookies. However, deselecting functional and advertising cookies may lead to a restriction of the functionalities of our websites. Essential cookies are set automatically and cannot be deselected, otherwise the websites would not work properly.
 
+### 5\. Google Analytics
+This website uses functions of the web analysis service Google Analytics. The provider of this service is Google Ireland Limited (“Google”), Gordon House, Barrow Street, Dublin 4, Ireland.
```
Comment (BeateRixen) [inline content/legal-notices/privacy-notice.md]: This website: This is strange as it also refers to the company website, doesn´t it?

File: content/legal-notices/privacy-notice.md [hunk 2]
```diff
@@ -95,8 +95,29 @@ We use the following types of cookies on our websites and online services:
...
 
-### 5\. How do we share your data and how is it transferred to third countries?
+Google Analytics enables the website operator to analyze the behavior patterns of website visitors. To that end, the website operator receives a variety of user data, such as pages accessed, time spent on the page, the utilized operating system and the user’s origin. This data is assigned to the respective end device of the user. An assignment to a user-ID does not take place.
+
+Furthermore, Google Analytics allows us to record your mouse and scroll movements and clicks, among other things. Google Analytics uses various modeling approaches to augment the collected data sets and uses machine learning technologies in data analysis.
+
+Google Analytics uses technologies that make the recognition of the user for the purpose of analyzing the user behavior patterns (e.g., cookies or device fingerprinting). The website use information recorded by Google is, as a rule transferred to a Google server in the United States, where it is stored.
```
Comment (BeateRixen) [inline content/legal-notices/privacy-notice.md]:
```suggestion
Google Analytics uses technologies that make the recognition of the user for the purpose of analyzing the user behavior patterns (for example, cookies or device fingerprinting). The website use information recorded by Google is, as a rule transferred to a Google server in the United States, where it is stored.
```

File: content/legal-notices/privacy-notice.md [hunk 3]
```diff
@@ -95,8 +95,29 @@ We use the following types of cookies on our websites and online services:
...
+
+The use of these services occurs on the basis of your consent pursuant to Art. 6(1)(a) GDPR and § 25(1) TTDSG. You may revoke your consent at any time.
+
+Data transmission to the US is based on the Standard Contractual Clauses (SCC) of the European Commission. Details can be found here: https://privacy.google.com/businesses/controllerterms/mccs/.
+
+#### Browser plug-in
```
Comment (BeateRixen) [inline content/legal-notices/privacy-notice.md]:
```suggestion
#### Browser plugin
```

File: content/legal-notices/privacy-notice.md [hunk 4]
```diff
@@ -95,8 +95,29 @@ We use the following types of cookies on our websites and online services:
...
 
+### 5\. Google Analytics
+This website uses functions of the web analysis service Google Analytics. The provider of this service is Google Ireland Limited (“Google”), Gordon House, Barrow Street, Dublin 4, Ireland.
 
-### 5\. How do we share your data and how is it transferred to third countries?
+Google Analytics enables the website operator to analyze the behavior patterns of website visitors. To that end, the website operator receives a variety of user data, such as pages accessed, time spent on the page, the utilized operating system and the user’s origin. This data is assigned to the respective end device of the user. An assignment to a user-ID does not take place.
```
Comment (BeateRixen) [inline content/legal-notices/privacy-notice.md]:
```suggestion
Google Analytics enables the website operator to analyze the behavior patterns of website visitors. To that end, the website operator receives a variety of user data, such as pages accessed, time spent on the page, the utilized operating system and the user’s origin. This data is assigned to the respective end device of the user. An assignment to a user ID does not take place.
```

**Comment by BeateRixen** [comment]
@JanePorter I adapted the spelling in 3 places but there is one openquestion where you need to look into.

File: content/legal-notices/privacy-notice.md (hunk 1 above)
Comment (JanePorter) [reply content/legal-notices/privacy-notice.md]: ha! got your point.  Yes, we should change it to 'The Cumulocity website'

Comment (BeateRixen) [reply content/legal-notices/privacy-notice.md]: But it is also true for the documentation website. So perhaps "The Cumulocity websites use ..."?

**Review by pink-mongoose — APPROVED** [review]

**Review by pink-mongoose — APPROVED** [review]

File: content/legal-notices/privacy-notice.md (hunk 1 above)
Comment (BeateRixen) [inline content/legal-notices/privacy-notice.md]:
```suggestion
The Cumulocity websites use functions of the web analysis service Google Analytics. The provider of this service is Google Ireland Limited (“Google”), Gordon House, Barrow Street, Dublin 4, Ireland.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**